.do/db-cluster/variables.tf

```diff
@@ -16,6 +16,10 @@ variable "k8s_cluster_id" {
   description = "The id of the k8s cluster where db connections will come from."
   type = string
   default = "REPLACE_ME"
```

Comment:
You could also refactor this to have as input the cluster name, and then use a [data source](https://registry.terraform.io/providers/digitalocean/digitalocean/latest/docs/data-sources/kubernetes_cluster) to get the cluster, from which you can extract its data, namely, its id

Reply:
Thought it would be preferable to keep using the same strategy.

Reply:
Implemented.